content/ReleaseNotes/TrueCommand/2.0-Beta.md

```diff
@@ -0,0 +1,115 @@
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1635'>TC-1635</a>] -         TC Services/Install scripts
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1677'>TC-1677</a>] -         Confirm Dialog -- need enhancements for data protection
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1709'>TC-1709</a>] -         Add Disable for usage reporting to admin page
+</li>
+</ul>
+
+### Improvement
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1577'>TC-1577</a>] -         UI suggestions / issues
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1578'>TC-1578</a>] -         iSCSI Delete option
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1624'>TC-1624</a>] -         Progress indicatios during cluster dataset creation
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1625'>TC-1625</a>] -         Cluster dataset deletion
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1626'>TC-1626</a>] -         Add navigation shortcut to Cluster Manager
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1643'>TC-1643</a>] -         NAS Gluster APIs Update
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1649'>TC-1649</a>] -         IP mapping fixes and other data flow issues
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1656'>TC-1656</a>] -         Config Backup Dialog Box has no tooltips for icons
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1710'>TC-1710</a>] -         Cleanup Dashboard card: Rework network/disk meters.
+</li>
+</ul>
+
+
+
+### Bug Fixes
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1602'>TC-1602</a>] -         Can't connect to TrueNAS-12.0-U2.1 with HTTP to HTTPS redirect enabled
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1640'>TC-1640</a>] -         Incorrect TrueCommand Email Alert Subject Lines
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1650'>TC-1650</a>] -         Save/Cancel buttons in configuration dialog not active
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1670'>TC-1670</a>] -         Description is not saved when creating/editing a Report
+</li>
+</ul>
+
+### Known Issues
+
+<body class="ql-editor ql-editor-view" style="font-size:14px;">
+    <html>
+        <body>
+            <table width="100%">
+                <thead><tr><th>Key</th><th>Summary</th><th>Workaround</th></tr></thead>
+                <tbody>
+                   <tr>
+                      <td><a href="https://jira.ixsystems.com/browse/NAS-110413" target="_blank">NAS-110413</a></td>
+                      <td>The UI is misreading the schedule for a Snapshot Task from True Command.</td>
+                      <td></td>
+                  </tr>
+                  <tr>
+                      <td></td>
+                      <td>Cluster Volumes.</td>
+                      <td> This is a brand new feature and is still being polished.</td>
```

Comment:
and is still being polished. Please report bugs <here>

Reply:
Added